config/rm/templates/database.mem.properties

```diff
@@ -0,0 +1,35 @@
+# Database connection settings.
+# To change the database provider, change the driver (hibernate.connection.driver_class),
+# the URL (hibernate.connection.url), and don't forget the dialect (hibernate.dialect)
+
+hibernate.connection.driver_class=org.hsqldb.jdbc.JDBCDriver
+hibernate.connection.url=jdbc:hsqldb:mem:rm;hsqldb.tx=mvcc;hsqldb.lob_file_scale=1
```

Comment:
Can you double check this line? The syntax highlighting suggests that there is an issue.